package.json

```diff
@@ -2,20 +2,19 @@
   "name": "iam-client-lib",
   "version": "8.0.0-alpha.4",
   "description": "Library for Decentralized Identity and Access Management",
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/src/index.d.ts",
+  "main": "lib/index.js",
+  "module": "lib.esm/index.js",
+  "types": "lib/src/index.d.ts",
   "files": [
-    "dist"
+    "lib",
+    "lib.esm"
   ],
   "scripts": {
     "lint": "eslint ./src/**/*.ts ./e2e/**/*.ts",
-    "prebuild": "rimraf dist ethers",
-    "build": "npm run build:contracts && npm run build:ts && copyfiles \"ethers/**/*.d.ts\" dist",
```

Comment:
Is `copyfiles \"ethers/**/*.d.ts\" dist` no longer needed? If it is not needed anymore, perhaps `copyfiles` can be removed as a dev dependency

Reply:
yes, contract definitions are part of the code, so they are included in build. I will remove `copyfiles`. good catch. may be we can use https://www.npmjs.com/package/npm-check